### ISSUE TITLE
How to do "cherry picking"

### ISSUE THREAD

**Issue by adriatic** [aurelia-ui-toolkits/screenshots#8]

**Comment by JeroenVinke** [comment]
![image](https://cloud.githubusercontent.com/assets/2189477/13327666/34ff05e0-dbeb-11e5-99e2-b24cae747aae.png)

Sourcetree (I use sourcetree instead of github desktop client) allows you to select a commit, right click and click "cherry pick". 

But this is the git command: https://git-scm.com/docs/git-cherry-pick

I believe you need to you `git cherry-pick <commit hash>`

**Comment by adriatic** [comment]
Thank you. I thought initially that the name "cherry-picking" is your own "slang" for a trick that you created yourself. Had I known that this is a git feature, I would look it up without bugging you.

**Comment by JeroenVinke** [comment]
that makes sense :)

**Comment by adriatic** [comment]
It's also interesting that you use Sourcetree despite your stated preference for GitHub client

**Comment by JeroenVinke** [comment]
Github desktop client is more simplistic while sourcetree contains way more features (but is also a bit more hard to follow sometimes)
